src/main/resources/css/settingsv2.css

```diff
@@ -0,0 +1,84 @@
+.settingsBorder{
```

Comment:
do we really need a .css file? Some attributes like hbox-spacing, padding can also be set in the scene builder directly.

Reply:
It is still necessary but I moved some of the spacing and etc to scenebuilder